appservice/src/intTest/HibernateStatsTest.md

```diff
@@ -4,18 +4,18 @@
 | ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---
 | GetUserWithoutPrivateDataFirst| 0| 0| 0| 0| 0| 0| 1| 0| 0| 0| 1| 0| 1| 0| 1| 0| 0| 0| 0| 0| 0| 0| 0| 3| 3| 1| 1
 | GetUserWithoutPrivateDataSecond| 0| 0| 0| 0| 0| 0| 1| 0| 0| 0| 1| 0| 1| 0| 1| 0| 0| 0| 0| 0| 0| 0| 0| 2| 2| 1| 1
-| GetUserWithPrivateDataFirst| 0| 7| 15| 0| 0| 0| 1| 0| 6| 0| 65| 0| 3| 0| 22| 0| 0| 0| 7| 2| 0| 0| 0| 2| 2| 3| 3
-| GetUserWithPrivateDataSecond| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 59| 0| 3| 0| 16| 0| 0| 0| 7| 5| 0| 0| 0| 2| 2| 3| 3
-| GetMessagesFirst| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 11| 28| 0| 0| 0| 2| 2| 67| 67
-| GetMessagesSecond| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 11| 36| 0| 0| 0| 2| 2| 67| 67
-| GetDayActivityOverviewsFirst| 0| 4| 11| 0| 0| 0| 1| 0| 3| 0| 34| 0| 2| 0| 11| 0| 0| 0| 2| 1| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityOverviewsSecond| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 31| 0| 2| 0| 6| 0| 0| 0| 1| 2| 0| 0| 0| 2| 2| 2| 2
-| GetWeekActivityOverviewsFirst| 0| 6| 26| 0| 0| 0| 1| 0| 3| 0| 93| 0| 2| 0| 13| 0| 0| 0| 2| 1| 0| 0| 0| 2| 2| 2| 2
-| GetWeekActivityOverviewsSecond| 0| 3| 23| 0| 0| 0| 1| 0| 3| 0| 89| 0| 2| 0| 8| 0| 0| 0| 1| 2| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityOverviewsWithBuddiesFirst| 0| 7| 51| 0| 0| 0| 1| 0| 4| 0| 256| 0| 3| 0| 30| 0| 0| 0| 14| 5| 0| 0| 0| 2| 2| 3| 3
-| GetDayActivityOverviewsWithBuddiesSecond| 0| 5| 49| 0| 0| 0| 1| 0| 3| 0| 255| 0| 3| 0| 27| 0| 0| 0| 14| 3| 0| 0| 0| 2| 2| 3| 3
-| GetDayActivityDetailsLastFridayFirst| 0| 4| 9| 0| 0| 0| 1| 0| 3| 0| 27| 0| 2| 0| 11| 0| 0| 0| 2| 1| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityDetailsLastFridaySecond| 0| 1| 6| 0| 0| 0| 1| 0| 2| 0| 20| 0| 2| 0| 5| 0| 0| 0| 1| 1| 0| 0| 0| 2| 2| 2| 2
+| GetUserWithPrivateDataFirst| 0| 7| 15| 0| 0| 0| 1| 0| 6| 0| 69| 0| 3| 0| 26| 0| 0| 0| 9| 5| 0| 0| 0| 2| 2| 3| 3
```

Comment:
The number of queries has increased, is that acceptable?

Reply:
Not really, but a failure is worse.  I've logged a ticket to gain it back: https://jira.yona.nu/browse/YD-579